Microsoft.Toolkit/Diagnostics/Generated/ThrowHelper.Collection.tt

```diff
@@ -13,9 +13,9 @@ using Microsoft.Toolkit.Extensions;
 namespace Microsoft.Toolkit.Diagnostics
 {
     /// <summary>
-    /// Helper methods to throw exceptions
+    /// Helper methods to efficiently throw exceptions.
```

Comment:
Do we want to comment this specifically that these are internal and more nicely formatted specifically for the Guard APIs?

Reply:
@Sergio0694 ?

Reply:
Whoops, sorry, forgot to reply to this comment too! 😅

I've addressed this in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3346/commits/7c7098ada9d7e3f06c59f3afd2b722311556e236 with the other related changes (see conversation below) - all the internal methods are now in the generated `ThrowHelper` files in the `\Internals` folder, and there's a single `ThrowHelper` file that's in the `\Diagnostics` folder right next to the `Guard` APIs, which includes all the public throw helper APIs added in this PR.

Is the current structure alright or should I still add some comments to explain the files organization somewhere?

Reply:
I think its fine.